stable/velero/templates/crds.yaml

```diff
@@ -0,0 +1,4 @@
+{{- range $path, $bytes := .Files.Glob "crds/*.yaml" }}
+{{ $.Files.Get $path }}
+---
+{{- end }}
```

Comment:
Just needs new line at EOF

Reply:
Fixded in https://github.com/helm/charts/pull/19042/commits/8d01cb373607144576f32369684e5e5c058622f5